content/docs/components/serving/seldon.md

```diff
@@ -47,6 +47,115 @@ spec:
 
 Save the above resource and apply it with `kubectl`.
 
-## Examples
+## Simple Example
 
-Seldon provides a [large set of example notebooks](https://docs.seldon.io/projects/seldon-core/en/latest/examples/notebooks.html) showing how to run inference code for a wide range of machine learning toolkits.
+
+Create a new namespace:
+
+```
+kubectl create ns testseldon
+```
+
+Label that namespace so we can run inference tasks in it:
+
+```
+kubectl label namespace testseldon serving.kubeflow.org/inferenceservice=enabled
+```
+
+Create an istio gateway in that namespace named `kubeflow-gateway`:
+
+```
+cat <<EOF | kubectl create -f -
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: kubeflow-gateway
+  namespace: testseldon
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - hosts:
+    - '*'
+    port:
+      name: http
+      number: 80
+      protocol: HTTP
+EOF
+```
+
+Create an example `SeldonDeployment` with a dummy model:
+
+
+```
+cat <<EOF | kubectl create -n testseldon -f -
+apiVersion: machinelearning.seldon.io/v1
+kind: SeldonDeployment
+metadata:
+  name: seldon-model
+spec:
+  name: test-deployment
+  predictors:
+  - componentSpecs:
+    - spec:
+        containers:
+        - image: seldonio/mock_classifier_rest:1.3
+          name: classifier
+    graph:
+      children: []
+      endpoint:
+        type: REST
+      name: classifier
+      type: MODEL
+    name: example
+    replicas: 1
+EOF
+```
+
+Wait for state to become available:
+
+```
+kubectl get sdep seldon-model -n testseldon -o jsonpath='{.status.state}\n'
+```
+
+Port forward to the Istio gateway:
+
+```
+kubectl port-forward $(kubectl get pods -l istio=ingressgateway -n istio-system -o jsonpath='{.items[0].metadata.name}') -n istio-system 8004:80
+```
+
+Send a prediction request:
+
+```
+curl -s -d '{"data": {"ndarray":[[1.0, 2.0, 5.0]]}}'    -X POST http://localhost:8004/seldon/testseldon/seldon-model/api/v1.0/predictions    -H "Content-Type: application/json"
+```
+
+You should see a response:
+
+```
+{
+  "meta": {
+    "puid": "i2e1i8nq3lnttadd5i14gtu11j",
+    "tags": {
+    },
+    "routing": {
+    },
+    "requestPath": {
+      "classifier": "seldonio/mock_classifier_rest:1.3"
+    },
+    "metrics": []
+  },
+  "data": {
+    "names": ["proba"],
+    "ndarray": [[0.43782349911420193]]
+  }
+}
+```
+
+
+## Further Documentation
```

Comment:
For consistency within the page and with the Kubeflow style guide, change this heading to sentence case:

```
## Further documentation
```